yang/frr-nexthop.yang

```diff
@@ -61,7 +61,7 @@ module frr-nexthop {
     type union {
       type inet:ip-address;
       type string {
```

Comment:
Since this module specifies YANG 1.1, we should probably just use `type empty;` here instead of a zero length string (specified either way).

Reply:
Yes I can try that tomorrow, I am not sure right now that it will work without any code change.